daemons/gptp/common/ieee1588port.cpp

```diff
@@ -863,7 +864,8 @@ void IEEE1588Port::processEvent(Event e)
 			 (pow ((double)2, getAnnounceInterval()) * 1000000000.0));
 		break;
 	case FAULT_DETECTED:
-		XPTPD_INFO("Received FAULT_DETECTED event");
+        XPTPD_INFO("Received FAULT_DETECTED event");
+        setAsCapable(false);
```

Comment:
Ok, same problem as above. I have spaces instead of tabs on my vim. I rolled it back and kept these two lines as tabs.